src/main/java/bio/terra/common/db/DatabaseRetryUtils.java

```diff
@@ -45,7 +45,7 @@ private DatabaseRetryUtils() {}
   }
 
   /**
-   * Tests an exception to see if it is retryable
+   * Tests an exception to see if it is retryable.
```

Comment:
I would almost suppress this one globally. I like to use fragments in comments, and the  absence of a trailing period can have some meaning.

Reply:
Done